test/compilable/extra-files/ddocYear.html

```diff
@@ -492,7 +492,7 @@ <h1 class="module_name">ddocYear</h1>
   <section class="section ddoc_sections">
   <div class="ddoc_summary">
   <p class="para">
-    2016
+    2017

```

Comment:
Correct fix is here: https://github.com/dlang/dmd/pull/6391
Otherwise it'll break again next year.